pkg/package-server/apis/operators/packagemanifest.go

```diff
@@ -32,3 +38,34 @@ func CreateCSVDescription(csv *operatorsv1alpha1.ClusterServiceVersion) CSVDescr
 
 	return desc
 }
+
+// descriptionsForCRDs filters certain fields from provided API descriptions to reduce response size.
+func descriptionsForCRDs(crds []operatorsv1alpha1.CRDDescription) []operatorsv1alpha1.CRDDescription {
```

Comment:
Strip out `spec` and `status` descriptors, we don't need them.

Reply:
We really shouldn't remove fields that we've already published. Is there an actual bug if they remain?

Reply:
Anything we can do to decrease the JSON response size. Perhaps it would be better to add a query parameter for filtering these fields (since it's an aggragated apiserver so we have that control).

Reply:
I recall there was a question on forum-operator about these 2 fields and we did say it is legacy-only with intention to remove those fields.

Reply:
If we want to actually remove fields we need to bump the version of the api.

Reply:
If the fields remain but are nil, this is fine.